### ISSUE TITLE
Unable to run the dependency-check.bat command from C# 

### ISSUE THREAD

**Issue by shubhamdhanetia** [dependency-check/DependencyCheck#2981]
I have downloaded the dependency-check zip for windows and extracted it. 
Then I set the environment variable and run the dependency check command using the command prompt, It worked fine.
For the first time, it took some time and installed the NVD Databases.

I am using Windows OS.

Now, I want to execute the command using the programming language C#.
So, when I try to execute the same command from C#, 
I get the error : Error: Could not find or load main class org.owasp.dependencycheck.App

Let me know if there is a proper way to call the command using C# and get the results.
Basically, I would be passing project directory, generate the report files, read the json file and return it to the user

**Comment by jeremylong** [comment]
I don't know what your code looks like nor do I have a good windows environment to test right now - but something like this should work:

```csharp
System.Diagnostics.Process process = new System.Diagnostics.Process();
System.Diagnostics.ProcessStartInfo startInfo = new System.Diagnostics.ProcessStartInfo();
startInfo.WindowStyle = System.Diagnostics.ProcessWindowStyle.Hidden;
startInfo.FileName = "cmd.exe";
startInfo.WorkingDirectory = @"c:\path\to\dependency-check\bin\";
startInfo.Arguments = "/C dependency-check.bat -s [path] -o [path]";
process.StartInfo = startInfo;
process.Start();
```

Of course fixing the WorkingDirectory and the paths passed into the call itself.

**Comment by shubhamdhanetia** [comment]
Hi, I was able to run the command but then I was getting this internal error instead of generating the report.

"Error: Could not find or load main class org.owasp.dependencycheck.App"

Now, In windows, I am running the shell script from the docker image documentation using the git bash
and I am getting a docker reference error.

**Comment by shubhamdhanetia** [comment]
Ok fixed now. Some command issues were there. Thank you.
